pom.xml

```diff
@@ -109,6 +115,45 @@
 						</execution>
 					</executions>
 				</plugin>
+				<plugin>
+				<groupId>org.jvnet.maven-antrun-extended-plugin</groupId>
+				<artifactId>maven-antrun-extended-plugin</artifactId>
+				<version>1.43</version>
+				<executions>
```

Comment:
This plugin seems to be duplicating the report genertion, see the next plugin

Reply:
Was this about the issue of the double compilation?

Reply:
The issue was that the large number of warnings make difficult to know what maven is doing if three are two places where the warnings are shown. Now that warnings are removed it is not a major problem. I'm inclined to remove the aggregate reporting for this project, but it can be postponed to the migration is completed. There is a 4th stage to check  and reorganize the reports. Go ahead as is.